### ISSUE TITLE
Problem with open modules

### ISSUE THREAD

**Issue by Roeverix** [WatForm/org.alloytools.alloy#6]
I wanted to use the util/integer module, but there is a problem,
```alloy 
open util/integer 
``` 
does not work. Error message: 
```
Could not import module: integer
```
I figured out, that i must do following.
```alloy 
open util/integer as i
``` 
Now it works. But just with the builded solution from Burkhardt Renz. If i build the current solution from the branch "dash" i get the following syntax error:
```
The name "isEnabled" cannot be found.
```

Kind regards ,
Ronja Haas

**Comment by nancyday** [comment]
Can you send us the Dash module that caused this error?

**Comment by Roeverix** [comment]
I just took a Dash module that had worked before and put it in there (e.g. MusicalChair or BitCounter). I am still quite inexperienced with Dash and therefore wanted to test first how I can include other modules.

**Comment by nancyday** [comment]
Do you still have the problem after the updates we made to the dash branch?

**Comment by Roeverix** [comment]
Yes. I made some pictures to show the problem. Maybe they can help. In the last picture are the settings, but i didn´t changed them. So these should be the default settings. 

![int](https://user-images.githubusercontent.com/77026045/172819197-53f23035-7656-4e24-b309-9b3849400c9b.png)

![asi](https://user-images.githubusercontent.com/77026045/172819432-ee971f74-3885-4c04-936a-7f84038d5585.png)

![settings](https://user-images.githubusercontent.com/77026045/172819460-1cad661e-d867-45b3-876e-16ad45aa4395.png)
